### PR TITLE
fix(context-selector): added suppport for items as links

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/3870

Also adds a11y docs.

### CODE REVIEW

File: src/patternfly/components/ContextSelector/examples/context-selector.md [hunk 1]
```diff
@@ -316,8 +112,14 @@ import './context-selector.css'
 
 ## Documentation
 ### Accessibility
-Added after React implementation.
+| `aria-expanded="false"` | `.pf-c-context-selector__toggle` |  Indicates that the menu is hidden. |
```
Comment (srambach) [inline src/patternfly/components/ContextSelector/examples/context-selector.md:115]: I think you might need a header row here; it's not displaying as a table.

Comment (mcoker) [reply src/patternfly/components/ContextSelector/examples/context-selector.md:117]: @srambach thanks! Updated.

File: src/patternfly/components/ContextSelector/context-selector.scss [hunk 2]
```diff
@@ -54,6 +54,7 @@
   --pf-c-context-selector__menu-list--MaxHeight: #{pf-size-prem(200px)};
 
   // Menu item
+  --pf-c-context-selector__menu-list-item--Color: var(--pf-global--Color--dark-100);
```
Comment (srambach) [inline src/patternfly/components/ContextSelector/context-selector.scss:57]: Should `--pf-c-context-selector__menu-list-item--Color` be in the CSS variables section of the docs?

Comment (mcoker) [reply src/patternfly/components/ContextSelector/context-selector.scss:57]: @redallen can you confirm?

Comment (redallen) [reply src/patternfly/components/ContextSelector/context-selector.scss:57]: It should be in that section once `react-tokens` takes in the change.

**Comment by srambach** [comment]
Other than the variable, which sounds like it'll automatically show up, LGTM! 👍

**Comment by mcoker** [comment]
@srambach great! Would you mind smashing the review/approve button?

**Review by srambach — APPROVED** [review]
👍

File: src/patternfly/components/ContextSelector/__context-selector-menu-menu.hbs [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    {{#> context-selector-menu-list-item}}
+      AWS
+    {{/context-selector-menu-list-item}}
+  </li>
+  <li>
+    {{#> context-selector-menu-list-item}}
+      Azure
+    {{/context-selector-menu-list-item}}
+  </li>
+{{/context-selector-menu-menu}}
```
Comment (mattnolting) [inline src/patternfly/components/ContextSelector/__context-selector-menu-menu.hbs:72]:
```suggestion
{{/context-selector-menu-menu}}

```

File: src/patternfly/components/ContextSelector/examples/context-selector.md [hunk 4]
```diff
@@ -26,58 +26,7 @@ import './context-selector.css'
...
-          AWS
-        {{/context-selector-menu-menu-item}}
-      </li>
-      <li>
-        {{#> context-selector-menu-menu-item}}
-          Azure
-        {{/context-selector-menu-menu-item}}
-      </li>
-    {{/context-selector-menu-menu}}
+    {{> __context-selector-menu-menu}}
```
Comment (mattnolting) [inline src/patternfly/components/ContextSelector/examples/context-selector.md:29]: 👍👍👍

**Review by mattnolting — APPROVED** [review]
LGTM 👍

**Comment by mcoker** [comment]
@mattnolting updated!

**Pull request merged**